README.md

```diff
@@ -24,32 +72,43 @@ If this occurs, run the following to [fix Powershell execution policy](https://a
 Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned
 ```
 
-## Development server
+## Development Server
 
 Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`.
 The application will automatically reload if you change any of the source files.
 
 The APIs are proxied to avoid CSRF issues; see [proxy.conf.json](src/proxy.conf.json)
 
-A working ReefGuide Web API instances is also needed.
+## ReefGuide Web API Integration
+
+A working ReefGuide Web API instance is required to use the application.
+
+**Option 1: Use existing API instance**
+- Configure the `webApiUrl` in the environment file to point to your API instance
```

Comment:
Specify the full path to the environment file `src\environments\environment.development.ts`
it's not obvious, this is the first section where it's introduced.